packages/frontend/src/renderer/containers/widgets/joinCommunity/joinCommunity.basic.test.tsx

```diff
@@ -126,65 +126,40 @@ describe('join community', () => {
     expect(createUsernameTitle).toBeVisible()
   })
 
-  it('user rejoins to remembered community with certificate', async () => {
+  it('user rejoins to remembered community with user certificate', async () => {
     const factoryStore = (await prepareStore()).store
     const factory = await getFactory(factoryStore)
 
     const community = await factory.create<
       ReturnType<typeof communities.actions.addNewCommunity>['payload']
     >('Community')
+    await factory.create<
+      ReturnType<typeof identity.actions.addNewIdentity>['payload']
+    >('Identity', { id: community.id, nickname: 'alice1' })
 
-    const identityAlpha: Identity = {
-      id: community.id,
-      nickname: 'nickname',
-      hiddenService: {
-        onionAddress: '',
-        privateKey: ''
-      },
-      dmKeys: {
-        publicKey: '',
-        privateKey: ''
-      },
-      peerId: {
-        id: '',
-        pubKey: '',
-        privKey: ''
-      },
-      userCsr: null,
-      userCertificate: ''
-    }
+    factoryStore.dispatch(socketActions.setConnected())
 
-    const { store } = await prepareStore({
-      [StoreKeys.Socket]: {
-        ...new SocketState(),
-        isConnected: true
-      },
-      [StoreKeys.Modals]: {
-        ...new ModalsInitialState(),
-        [ModalName.joinCommunityModal]: { open: true }
-      },
-      [NectarStoreKeys.Communities]: factoryStore.getState().Communities,
-      [NectarStoreKeys.Identity]: {
-        ...new identity.State(),
-        identities: identityAdapter.setAll(
-          identityAdapter.getInitialState(),
-          [identityAlpha]
-        )
-      }
-    })
+    factoryStore.dispatch(modalsActions.openModal({
+      name: ModalName.joinCommunityModal
+    }))
+
+    factoryStore.dispatch(identity.actions.storeUserCertificate({
+      userCertificate: '',
+      communityId: community.id
+    }))
 
     const result1 = renderComponent(
```

Comment:
Calling another render function later in the test will cause a mess - especially when attaching nectar's sagas. There should always be just one `renderComponent` call in the test